src/main/java/org/tikv/cdc/RegionCDCClient.java

```diff
@@ -127,7 +128,7 @@ public void close() throws Exception {
       channel.shutdown();
     }
     try {
-      LOGGER.debug("awaitTermination (region: {})", region.getId());
```

Comment:
The original code is the right way to use the logging framework.
http://www.slf4j.org/faq.html#logging_performance